packages/backend/src/models/UserModel.ts

```diff
@@ -864,20 +864,17 @@ export class UserModel {
         return row.refresh_token;
     }
 
-    async getOpenIdIssuer(
-        email: string,
-    ): Promise<OpenIdIdentityIssuerType | undefined | null> {
-        const row = await this.database('emails')
+    async getOpenIdIssuers(email: string): Promise<OpenIdIdentityIssuerType[]> {
```

Comment:
now we return a list of issuers, and we check which ones are disabled on the service